cafebabel/static/js/script.js

```diff
@@ -26,6 +26,50 @@ Array.from(document.querySelectorAll('a')).forEach(a => {
     a.setAttribute('target', '_blank')
 })
 
+/* smooth scroll to anchor https://stackoverflow.com/a/17733311/6481285 */
+function animate(elem, style, unit, from, to, time, prop) {
```

Comment:
Remove parameters you don't change/use like `unit` and `prop` and simplify the code accordingly.

Reply:
ok !